src/components/bs5/select/select2.functions.js

```diff
@@ -0,0 +1,30 @@
+// Making select2 fields accessible by screen readers
+export function handleScreenReaderAccessibility () {
+    var selects = document.getElementsByClassName("select2-search__field");
```

Comment:
I looked more in Squiz. There is a whole shim css file for `qg-search-widget` in production.
https://www.qld.gov.au/_admin/?screen=parse_file&assetid=499153

Ideally even the css shouldn't exist in Squiz. Another approach is making another 'story' for select2 within select component, because the purpose of use fits in the same category. 

However considering that the long term plan is getting rid of select2, neither of these approaches are worth spending the effort and is out of the scope of this task.

For now I think we can create a JS file under https://www.qld.gov.au/_admin/?screen=details&assetid=499166 to address the accessibility issue.

Reply:
Hi @ghazal-salehi , 

It just should still be split from the generic select as its its own 'dom' element.  Don't mind if its planned to go away.

As it currently is we can't run storybook/chromatic accessibility testing on it external from the cms. Please do bring the qg-search-widget component instead so it can be updated without risk of regression. For the pr, do bring to others attention that its 'legacy' code and not your own. (i.e. first commit should be status quo, so we can see what you did to improve it)

Reply:
Yes agree @ghazal-salehi @duttonw, this should not be introduced into DS.  

Let's make the accessibility improvements directly in Squiz (extend the Squiz SHIM file) and keep select2 references isolated there, with a view to remove it entirely as me migrate into new DS.

Reply:
Thanks Steve.